### PR TITLE
Milestone: Work modal

### PR DESCRIPTION
**Pull request by kendoriddy**
- Add HTML code for mobile view of work modal
- Style mobile view of work modal
- Add HTML code for desktop view of work modal
- Style desktop view of work modal
- Fix linters errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -95,8 +95,76 @@ <h1 class="introduction">
...
+                  </li>
+                  <li>
+                    <a href="#" target="_blank">See Source
+                      <img class="outlinks" src="./assets/images/github-white.svg" alt="project-image" /></a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </div>
           <div class="card">
```
Comment (topeogunleye) [inline index.html:168]: - [ ] I kindly suggest that you add your projects and modals dynamically using javascript. 
- [ ] Kindly implement an array of objects to store your JavaScript objects containing the information for each project.

## EXAMPLE
```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

- [ ] I kindly suggest that you ensure the Modal popup data is populated with the data from the projects array.

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @kendoriddy, @samlorlah,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (kendoriddy) [reply index.html]: Thanks @topeogunleye. Suggestion implemented

**Review by omarsalem7 — APPROVED** [review]
Hi @kendoriddy, @topeogunleye 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**